Source/Foundation/bsfUtility/Prerequisites/BsStdHeaders.h

```diff
@@ -223,13 +223,18 @@ namespace bs
 		constexpr T& operator*() const { return *mPtr; }
 		constexpr T* operator->() const { return mPtr; }
 		constexpr T* get() const { return mPtr; }
+		constexpr bool operator<(const NativePtr& rhs) const noexcept {
```

Comment:
BSF compiles with exceptions disabled, so the noexcept keyword isn't necessary here.